raiden/raiden_event_handler.py

```diff
@@ -287,7 +287,7 @@ def handle_send_refundtransfer(
         raiden: "RaidenService",
         refund_transfer_event: SendRefundTransfer,
         message_queues: Dict[QueueIdentifier, List[Tuple[Message, Optional[AddressMetadata]]]],
-    ) -> None:
+    ) -> None:  # pragma: no unittest
```

Comment:
Not sure anyone really cares for these pragmas anymore.

Reply:
✅